features/command_line/bisect.feature

```diff
@@ -54,12 +54,12 @@ Feature: Bisect
       Bisect started using options: "--seed 1234"
       Running suite to find failures... (0.16755 seconds)
       Starting bisect with 1 failing example and 9 non-failing examples.
+      Checking that failures are order-dependent... failure is order-dependent
```

Comment:
Technically, we don't actually know that the failure is order dependent -- it merely appears to be.  But it could flap for other reasons (e.g. based on randomness or some other indeterminacy) and it could just happen to fail with all examples and pass when run on its own. I don't feel strongly about this, but phrasing this as `failure appears to be order-dependent` is more accurate.  Thoughts?